src/utils/index.js

```diff
@@ -0,0 +1,3 @@
+const isEmptyString = value => !value || (typeof value === 'string' && value.trim() === '') ;
+
+export default isEmptyString;
```

Comment:
shouldn't be a default export as there will be other util methods too

Reply:
Done